osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-smr500.json

```diff
@@ -1910,6 +1910,51 @@
           "access": "RW"
         }
       ]
+    },
+    {
+      "tag": "READ_MBUS_STATUS",
+      "description": "Read MBus Status",
+      "note": null,
+      "class-id": 4,
+      "version": 0,
+      "obis": "0.x.24.2.6.255",
+      "group": "ELECTRICITY",
```

Comment:
This object is not electricity related. My suggestion would be to use the groups as defined in the (D)SMR documents: Abstract objects, Electricity related objects, M-bus related objects and Miscellaneous objects. So in this case the group should be MBUS.

Reply:
Corrected and added a README file where a basic description is given of these possible values.